lib/fastlane/plugin/firebase_app_distribution/helper/firebase_exceptions.rb

```diff
@@ -0,0 +1,17 @@
+class FirebaseAppDistributionError < StandardError
```

Comment:
If all we will be doing is using the error messages in this class, it doesn't need to inherit from `StandardError`. Consider removing the inheritance and maying renaming  the class to `FirebaseAppDistributionErrorMessage` and the file to `firebase_app_distribution_error_message.rb`.

Or maybe we can just call it `ErrorMessage` so it's shorter 🤔 ? Up to you!